Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:14
+
+# Replace KEYCLOAK_URL with the relevant URL for your own Keycloak hosting
+ENV KEYCLOAK_URL="https://keycloak-int.data-marketplace.io/auth/realms/marketplace/protocol/openid-connect/userinfo"
```

Comment:
It would be best to not set the default value to our keycloak url

Reply:
Sure, I'll change this

Reply:
I've removed the keycloak URL